cmd/promtool/main.go

```diff
@@ -364,6 +382,55 @@ func (ls lintConfig) lintDuplicateRules() bool {
 	return ls.all || ls.duplicateRules
 }
 
+// Check server health.
+func CheckServerHealth(url string) error {
+	if url == "" {
+		url = "http://localhost:9090"
+	}
+
+	// Check Health
+	healthCheckResp, err := http.Get(url + "/-/healthy")
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "  FAILED:", err)
+		return err
+	} else if healthCheckResp.StatusCode != 200 {
+		return fmt.Errorf("health check failed: URL=%s, status=%d", url, healthCheckResp.StatusCode)
+	}
+
+	healthCheckBody, err := io.ReadAll(healthCheckResp.Body)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, "  FAILED:", err)
+		return err
+	}
+
+	fmt.Fprintf(os.Stderr, "  SUCCESS: %v", string(healthCheckBody))
+	fmt.Println()
```

Comment:
Do we need to print a blank line here? If this is actually required, can we club it with the print statement written right above this?